src/base/dof_map_constraints.C

```diff
@@ -702,7 +702,7 @@ private:
               for (unsigned int qp=0; qp<n_qp; qp++)
                 {
                   // solution at the quadrature point
-                  OutputNumber fineval = 0;
+                  OutputNumber fineval(0);
```

Comment:
>> gcc --version
gcc (Ubuntu 4.8.4-2ubuntu1~14.04) 4.8.4

@roystgnr were you able to build with --enable-complex without my fixes?